PVTPackage/source/MultiphaseSystem/DeadOilMultiphaseSystem.cpp

```diff
@@ -62,15 +92,43 @@ std::unique_ptr< DeadOilMultiphaseSystem > DeadOilMultiphaseSystem::build( const
                                                                            const std::vector< double > & surfaceDensities,
                                                                            const std::vector< double > & molarWeights )
 {
+  const bool containsOil = std::find( phases.cbegin(), phases.cend(), pvt::PHASE_TYPE::OIL ) != phases.cend();
+  const bool containsGas = std::find( phases.cbegin(), phases.cend(), pvt::PHASE_TYPE::GAS ) != phases.cend();
+  const bool containsWater = std::find( phases.cbegin(), phases.cend(), pvt::PHASE_TYPE::LIQUID_WATER_RICH ) != phases.cend();  
+
+  const bool isValid = containsOil && ( containsGas || containsWater );
+  if( !isValid )  
+  {    
+    const std::string msg = "Three types of DO systems are allowed: Oil-Water-Gas, Oil-Water, and Oil-Gas";
+    LOGERROR( msg );
```

Comment:
This `LOGERROR` kills the app, yes?

Reply:
Yes it kills the app from what I saw here:
https://github.com/GEOSX/PVTPackage/blob/d24a4886c01fc43d3583effc43b71b6f42034ed3/PVTPackage/source/Utils/Logger.cpp#L73-L83